### PR TITLE
fixes all deserialization problems except for mesos/services which re…

### PR DESCRIPTION
**Pull request by kchengsf**
…quires an sbrest change

### CODE REVIEW

File: kubernetes/architecture.json [hunk 1]
```diff
@@ -14,40 +11,6 @@
   "discoveryQuery": [
     "_exists_:kubernetes_cluster"
   ],
-  "propertyColumns": [
```
Comment (jrcamp) [inline kubernetes/architecture.json:17]: I can see why `listColumns` aren't needed in architecture views but is `propertyColumns` not used for the properties tab when you click on a node?

File: kubernetes/architecture.json [hunk 2]
```diff
@@ -3,9 +3,6 @@
   "unreleased": true,
   "type": "architectural",
   "displayName": "Architecture",
-  "requiredProperties": [
```
Comment (jrcamp) [inline kubernetes/architecture.json:6]: Is `requiredProperties: ['host']` just a default somewhere or was this not needed?

File: kubernetes/pods.json [hunk 3]
```diff
@@ -9,9 +9,6 @@
   "requiredProperties": [
     "kubernetes_pod_name"
   ],
-  "filterProperties": [
```
Comment (jrcamp) [inline kubernetes/pods.json:12]: Are you sure this should be removed? Note that it's different than `requiredProperties`.

**Comment by kchengsf** [comment]
requiredProperties, propertyColumns, and listProperties are only allowed in elemental views.  I think for architectural views there does not exist any way to customize the Properties view, since they could all be completely different things.  It just does a property dump.

**Comment by jrcamp** [comment]
In signalview I just removed `propertyColumns` from an architectural view and it resulted in the properties tab now being empty.

**Comment by kchengsf** [comment]
Hmm interesting you're right.  I was basing it on the AWS Architecture one which also exhibits this empty Properties behavior.  I'll fix it on SBRest

**Pull request closed without merging**